web/pom.xml

```diff
@@ -111,14 +111,54 @@
             </dependencies>
         </profile>
         <profile>
-            <id>weblogic</id>
+            <id>weblogic-12.2.1</id>
             <dependencies>
                 <dependency>
+                    <groupId>com.oracle.weblogic</groupId>
+                    <artifactId>weblogic-server-pom</artifactId>
+                    <version>12.1.3-0-0</version>
+                    <type>pom</type>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
                     <groupId>org.jboss.arquillian.container</groupId>
-                    <artifactId>arquillian-wls-remote-12.1</artifactId>
+                    <artifactId>arquillian-wls-remote-12.1.2</artifactId>
                     <version>1.0.0.Alpha3</version>
                     <scope>test</scope>
+                </dependency> 
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.18.1</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <arquillian.launch>weblogic-12.2.1</arquillian.launch>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        
+        <profile>
+            <id>weblogic-12.1.3</id>
+            <dependencies>
+                <dependency>
+                    <groupId>com.oracle.weblogic</groupId>
+                    <artifactId>weblogic-server-pom</artifactId>
+                    <version>12.1.3-0-0</version>
+                    <type>pom</type>
+                    <scope>provided</scope>
                 </dependency>
+                <dependency>
+                    <groupId>org.jboss.arquillian.container</groupId>
+                    <artifactId>arquillian-wls-remote-12.1.2</artifactId>
```

Comment:
I have no idea why, but `arquillian-wls-remote-12.1.2` fails while `arquillian-wls-remote-12.1` works ok. Any clue?
Rest works like a charm, together with weblogic.xml and JAXRS injection.

```
<Feb 3, 2015 8:17:29 AM GMT> <Warning> <Deployer> <BEA-149003> <Unable to access application source information in "/tmp/arquillian1827025331194689942e4ca8901-6054-4327-9565-bd0258476e9d.war/e4ca8901-6054-4327-9565-bd0258476e9d.war" for application "e4ca8901-6054-4327-9565-bd0258476e9d". The specific error is: No application files exist.> 
<Feb 3, 2015 8:17:29 AM GMT> <Warning> <Deployer> <BEA-149124> <Failures were detected while initiating deploy task for application "e4ca8901-6054-4327-9565-bd0258476e9d". Error is: "[Deployer:149003]Unable to access application source information in "/tmp/arquillian1827025331194689942e4ca8901-6054-4327-9565-bd0258476e9d.war/e4ca8901-6054-4327-9565-bd0258476e9d.war" for application "e4ca8901-6054-4327-9565-bd0258476e9d". The specific error is: No application files exist."> 
<Feb 3, 2015 8:17:29 AM GMT> <Warning> <RMI> <BEA-080003> <A RuntimeException was generated by the RMI server: javax.management.remote.rmi.RMIConnectionImpl.invoke(Ljavax.management.ObjectName;Ljava.lang.String;Ljava.rmi.MarshalledObject;[Ljava.lang.String;Ljavax.security.auth.Subject;)
 javax.management.RuntimeMBeanException: java.lang.RuntimeException: [Deployer:149003]Unable to access application source information in "/tmp/arquillian1827025331194689942e4ca8901-6054-4327-9565-bd0258476e9d.war/e4ca8901-6054-4327-9565-bd0258476e9d.war" for application "e4ca8901-6054-4327-9565-bd0258476e9d". The specific error is: No application files exist..
javax.management.RuntimeMBeanException: java.lang.RuntimeException: [Deployer:149003]Unable to access application source information in "/tmp/arquillian1827025331194689942e4ca8901-6054-4327-9565-bd0258476e9d.war/e4ca8901-6054-4327-9565-bd0258476e9d.war" for application "e4ca8901-6054-4327-9565-bd0258476e9d". The specific error is: No application files exist.
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.rethrow(DefaultMBeanServerInterceptor.java:839)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.rethrowMaybeMBeanException(DefaultMBeanServerInterceptor.java:852)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:821)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at weblogic.management.mbeanservers.domainruntime.internal.FederatedMBeanServerInterceptor.invoke(FederatedMBeanServerInterceptor.java:377)
	Truncated. see log file for complete stacktrace
Caused By: java.lang.RuntimeException: [Deployer:149003]Unable to access application source information in "/tmp/arquillian1827025331194689942e4ca8901-6054-4327-9565-bd0258476e9d.war/e4ca8901-6054-4327-9565-bd0258476e9d.war" for application "e4ca8901-6054-4327-9565-bd0258476e9d". The specific error is: No application files exist.
	at weblogic.deploy.internal.adminserver.operations.OperationHelper.validateSource(OperationHelper.java:357)
	at weblogic.deploy.internal.adminserver.operations.OperationHelper.getArchiveVersionIdFromSource(OperationHelper.java:167)
	at weblogic.deploy.internal.adminserver.operations.OperationHelper.getAndValidateVersionIdWithSrc(OperationHelper.java:205)
	at weblogic.deploy.internal.adminserver.operations.ActivateOperation.updateConfiguration(ActivateOperation.java:50)
	at weblogic.deploy.internal.adminserver.operations.DeployOperation.updateConfiguration(DeployOperation.java:106)
	Truncated. see log file for complete stacktrace
> 
<Feb 3, 2015 8:17:29 AM GMT> <Warning> <RMI> <BEA-080003> <A RuntimeException was generated by the RMI server: javax.management.remote.rmi.RMIConnectionImpl.invoke(Ljavax.management.ObjectName;Ljava.lang.String;Ljava.rmi.MarshalledObject;[Ljava.lang.String;Ljavax.security.auth.Subject;)
 javax.management.RuntimeOperationsException.
javax.management.RuntimeOperationsException
	at javax.management.remote.rmi.RMIConnectionImpl.checkNonNull(RMIConnectionImpl.java:1665)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:800)
	at javax.management.remote.rmi.RMIConnectionImpl_WLSkel.invoke(Unknown Source)
	at weblogic.rmi.internal.BasicServerRef.invoke(BasicServerRef.java:701)
	at weblogic.rmi.internal.BasicServerRef$1.run(BasicServerRef.java:527)
	Truncated. see log file for complete stacktrace
Caused By: java.lang.IllegalArgumentException: ObjectName must not be null
	at javax.management.remote.rmi.RMIConnectionImpl.checkNonNull(RMIConnectionImpl.java:1663)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:800)
	at javax.management.remote.rmi.RMIConnectionImpl_WLSkel.invoke(Unknown Source)
	at weblogic.rmi.internal.BasicServerRef.invoke(BasicServerRef.java:701)
	at weblogic.rmi.internal.BasicServerRef$1.run(BasicServerRef.java:527)
	Truncated. see log file for complete stacktrace
```